xc/priority_auths.lua

```diff
@@ -8,40 +8,49 @@ local _M = { }
 -- @return true if authorized, false if denied, nil if unknown
 -- @return reason why the authorization is denied (optional, required only when denied)
 function _M.authorize(service_id, credentials, metric)
-  local redis_pub, ok_pub = redis_pool.acquire()
+  local redis_pub, ok_pub, err = redis_pool.acquire()
 
   if not ok_pub then
+    ngx.log(ngx.WARN, "[priority auths] couldn't connect to redis pub: ", err)
```

Comment:
All these calls look the same: `ngx.log(ngx.WARN, error_prefix, err)`. How about extracting a `log_warning(error_prefix, err)` local method or something similar?

Also, I think that extracting all these error prefixes to a local constant would improve readability. Something like: `local errors = { redis_pub_connect: "[priority_auths] couldn't connect to redis pub: ", redis_sub_connect: "...", ... }`

What do you think?